tests/dialects/test_snowflake.py

```diff
@@ -426,29 +426,18 @@ def test_null_treatment(self):
     def test_sample(self):
```

Comment:
We used to convert `SAMPLE` to `TABLESAMPLE`, [which are both synonymous in Snowflake](https://docs.snowflake.com/en/sql-reference/constructs/sample):

> SAMPLE and TABLESAMPLE are synonymous and can be used interchangeably.

With the change in the parser related to `kind` (see `_parse_table_sample`), we now actually use whatever the token's text was, so I adapted these tests accordingly.